go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/agext/levenshtein v1.2.3 // indirect
-	github.com/cloudflare/cloudflare-go v0.101.0
+	github.com/cloudflare/cloudflare-go v0.103.0
```

Comment:
let's remove this as well; we have dependabot bump the versions so individual PRs don't get queued waiting for this to land.

Reply:
Sounds good! Thanks for that context.